source/common/upstream/eds.cc

```diff
@@ -42,7 +42,7 @@ EdsClusterImpl::EdsClusterImpl(const envoy::api::v2::Cluster& cluster, Runtime::
 void EdsClusterImpl::startPreInit() { subscription_->start({cluster_name_}, *this); }
 
 void EdsClusterImpl::onConfigUpdate(const ResourceVector& resources) {
-  std::vector<HostSharedPtr> new_hosts;
+  std::vector<std::vector<HostSharedPtr>> new_hosts(4);  // Large enough to avoid most resizes.
```

Comment:
Good use of https://github.com/abseil/abseil-cpp/blob/master/absl/container/inlined_vector.h ?

Reply:
Fixed by changing around the memory model (which I needed to do in case there were priority levels with no updates)